certificate-policy.md

```diff
@@ -600,90 +599,117 @@ The CA SHALL disclose all Cross Certificates that identify the CA as the Subject
+
+CAs SHALL NOT issue Certificates containing any Fully Qualified Domain Names that are not under the gTLDs for DotGov or DotMil.  
 
-Within 30 days after ICANN has approved a new gTLD for operation, as evidenced by publication of a contract with the gTLD operator on [www.ICANN.org] each CA MUST (1) compare the new gTLD against the CA's records of valid certificates and (2) cease issuing Certificates containing a Domain Name that includes the new gTLD until after the CA has first verified the Subscriber's control over or exclusive right to use the Domain Name in accordance with Section 3.2.2.4.
+Approval of certificate applications requires successful completion of validation per Section 3.2. 
+
+In accordance with Section 5.5.2, all CAs SHALL maintain an internal database of all previously revoked Certificates and previously rejected certificate requests due to suspected phishing or other fraudulent usage or concerns. All CAs SHALL use this information to identify subsequent suspicious certificate requests and MAY use it as the basis for rejecting a certificate request.
 
-Within 120 days after the publication of a contract for a new gTLD is published on [www.icann.org], CAs MUST revoke each Certificate containing a Domain Name that includes the new gTLD unless the Subscriber is either the Domain Name Registrant or can demonstrate control over the Domain Name.
 
 ### 4.2.3 Time to process certificate applications
-Certificate applications SHALL be processed and a certificate issued within 30 days of identity verification.
+Certificate applications SHALL be processed and a certificate issued within 30 days of the first identity verification.
 
 ## 4.3 Certificate issuance
 
 ### 4.3.1 CA actions during certificate issuance
-Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
+Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.  Issuance of a CA certificate by the Root CA SHALL require written authorization by the Policy Authority.  
+
+All end entity certificates for Domain Validation SSL and Organizational Validation SSL SHALL assert a Certificate Transparency (CT) Signed Certificate Timestamp (SCT) via the x509v3 certificate extension.  The Issuing CA SHALL submit a precertificate to a minimum of two Certificate Transparency Logs.  Information included in the end entity certificates SHALL NOT be redacted prior to submission to the Certificate Transparency Logs.  At a minimum, the Issuing CA SHALL request and include two SCTs in the x509v3 certificate extension:
```

Comment:
@LarryFrank I was briefly confused myself, because it discusses EV in the second paragraph, but that's only to mention that EV certificates have to meet the definition in this document. The document is careful not to limit its scope to EV certificates.

I reached out to Chrome representatives and confirmed that the May 2016 document is their broad definition of `CT Qualified`, applies to Symantec, and is the current policy that would apply to any subsequent CT requirements for issued certificates.  (While we could probably recommend edits to make it even more clear, the document as written does match this interpretation.)

They obviously may (and I'd guess probably will) revise it before the April 2018 deadline and to get into greater alignment with whatever Mozilla goes with. But it is current policy, and there's no reason to think that certificate life time wouldn't continue to be correlated with a required number of SCTs.